---
layout: post
title:  "Biolit 02: protein targeting"
date:   2014-09-11 12:00:00
author: ericminikel
tag: bbs-230
location: Boston, MA
---

*These are my notes for week 02 of Harvard's BBS 230 "Analysis of the biological literature" course (September 9-11, 2014). This includes my own reading and analysis of the papers, as well as my notes from discussion section and lecture.*

### [Bankaitis 1986]: Isolation of yeast mutants defective in protein targeting to the vacuole

#### Background

In 1986, it was known that different proteins localized to different parts of the cell, but no one had any idea how proteins were targeted to different subcellular locations. This was a hot research area. Bankaitis sought to create a screen that could identify proteins involved in trafficking to the vacuole.

#### Terms

+ **Vacuole** is the yeast equivalent of the mammalian lysosome.
+ **Periplasm** is the space between the yeast inner membrane and cell wall.
+ **Zymogen** means an inactive proenzyme.
+ **CPY** is carboxypeptidase Y, a serine protease which was known to be targeted to the lysosome.
    + *PRC1* is the yeast gene which encodes CPY.
    + proCPY is the zymogen form of CPY. The paper states that when it reaches the vacuole, an 8,000 kDa prodomain is cleaved by a protease called PEP4 to yield mCPY (mature CPY). This is almost certainly a typo. The [entire amino acid sequence](http://www.uniprot.org/uniprot/P00729) of CPY is only 532 residues long, so there is no way the prodomain could be 8,000 kDa. More likely they meant 8 kDa / 8,000 Da.
+ **Invertase** is an enzyme that breaks down sucrose. It is essential for yeast to survive when sucrose is their sole carbon source.
    + *SUC2* is the yeast gene which encodes invertase.

#### What they did and why

The authors sought to create a screen which would allow them to select for mutants that are deficient in targeting their proteins to the vacuole. They used yeast strains deficient in endogenous *SUC2* so that, by default, they were unable to survive on a diet of sucrose alone. Then they transformed these yeast with a construct expressing a CPY-Invertase fusion protein which they call pCYI433. It contains 433 N-terminal amino acids of proCPY (about 80% of the total peptide sequence) fused to the entirety of invertase. By default, this fusion protein is targeted to the vacuole due to its CPY domain. If and only if the cell has a mutation which precludes effective targeting of proteins to the vacuole, the fusion protein will end up in the periplasm instead. Invertase can only function in the periplasm, so when these yeast are grown on media with sucrose as the sole carbon source, *only* the mutants that lack effective vacuole targeting will survive. This paper develops this screen, uses it to isolate several mutants, and then characterizes them.

#### Step by step

+ Figure 1A shows the construct used to express pCYI433 (explained above).
+ The authors expressed pCYI433 initially in the yeast strain SEY2101 (see table below). In Figure 1B, they characterize the behavior of this fusion protein in this yeast strain. The difference between the lanes with and without tunicamycin (a glycosylation inhibitor) shows that the fusion protein gets glycosylated (as expected, since CPY is glycosylated), and the difference between the *PEP4+* and *&Delta;pep4* genotypes shows that (unlike wild-type CPY) the pCYI433 prodomain is *not* being cleaved in the vacuole. Together, the facts that they can detect pCYI433, that it seems to undergo some normal processing (glycosylation) but is not cleaved (thus we don't need to worry about whether its invertase activity changes upon cleavage) all validate that this protein is sufficiently well-behaved that we can study it.
+ Next they wished to confirm their expectaton that pCYI433 would be targeted to the vacuole. To do this, they measure enzymatic activity of whole cell lysate versus vacuole fractions isolated via centrifugation. 

+ *PRC1* is the gene for PCY, *SUC2* is the gene for invertase. The authors created a protein which fuses invertase to 433 amino acids from the N terminus of the prodomain of CPY (proCPY). The use of only part of the prodomain means that the fusion protein does not get processed in the vacuole.
+ CPY normally comes in different forms. In its p1 form found in the ER and Golgi, it is partially glycosylated (a total of 67 kDa of glycan chains). p2 is found in the Golgi and has 69 kDa of glycan chains. When CPY arrives in the vacuole, an 8000 kDa prodomain is cleaved by PEP4, yielding mCPY (mature CPY). 
+ These yeast are deficient in endogenous *SUC2*. In order for yeast to live, invertase must be expressed and must locate in the periplasm. So if CPY-invertase fusion protein is correctly trafficked to the vacuole, then there is no invertase in the periplasm and the yeast die. Only mutants that are defective in vacuole trafficking can live.
+ They see no difference in something between *PEP4+* and *&Delta;pep4* genotypes 
+ Develops a fusion protein assay to be able to identify mutants with defects in trafficking to the vacuole
+ CPY is known to go to the vacuole
+ Invertase normally goes to periplasm; by fusing it to CPY it goes to vacuole
+ Mutants defective in vacuole targeting fall into 8 complementation groups. This suggests that the pathway can be disrupted at many different steps.
+ Figure 1B shows that (A) when tunicamycin is used to inhibit glycosylation, the fusion protein is lighter, as expected, (2) the fusion protein is stable whether within (*PEP4+*) or without (*&Delta;pep4*) the vacuole, which makes it possible to study it. The fuzziness of the bands might reflect some amount of proteolysis.
+ Zymogen means an inactive proenzyme. Wild-type yeast have a partially glycosylated zymogen
+ Their mutants expressing pCYI433 but lacking endogenous *SUC2* can survive on glucose no matter what, but can survive on sucrose if and only if they have a defect in targeting pCYI433 to the vacuole, allowing invertase to reach the periplasm.
+ They used this construct to identify mutant strains. Then they wanted to see if these strains also had defects in trafficking of wild-type CPY, so they removed the pCYI433 construct ("cured" the strains of it) and then made them *PRC1+* (i.e. they re-introduced wild-type CPY) and studied the localization and processing (glycosylation, proteolysis) of CPY.
+ To be sure that the mutants differed in mutations in other genes, they (A) sequenced them and showed that the pCYI433 construct was the same in all, and (B) did complementation group and showed 8 complementation groups, so at most 1 of those could be a mutation in the construct.
+ They create vacuole fractions (Table 1) and show that &alpha;-mannosidase (a control known to go entirely to the vacuole) and that the fusion protein are both found in the same fraction - i.e. 41-43% of each is found in the vacuole fraction. This is <100% because some is in the ER/Golgi on its way to the vacuole, and some is lost in the lysis prep, etc. So this 41% recovery figure may actually reflect 100% trafficking to the vacuole, and even if it didn't, it might be a high enough percentage to give a growth phenotype on sucrose. This is considered evidence that the fusion protein localizes to the vacuole, and not to one of the places those other proteins go (&alpha;-glucosidase is cytosolic, and NADPH-cytochrome c reductase is found in the ER). It would be nice to have >1 replicate here, though, and to know that you get *consistently* 41%. Note that the enzymes are being quantified by *enzymatic activity*. It would have been nice to have wild-type CPY as a comparison, but because it is a protease, measuring its activity would have been harder than measuring the activity of these other enzymes.
+ In Figure 2, they are asking how the mutants they have isolated affect the time-course trafficking of CPY. It is a mystery why in the first row the amount of mCPY increases between 15' and 30', yet no pCPY is seen at 15'. So where does the *more* of mCPY come from at 30'? Also they claim that the trafficking to the vacuole has a half-life of 6 minutes, implying ~3/4 of it should already be there by 15'. The authors would have done well to include a 6 minute timepoint in their pulse-chase. There is no loading control; it may just be there was more protein in the 30' sample. (Perhaps they left the 15 minute sample on the bench while continuing the pulse-chase).
+ Importantly, though, the blot does show that the pCPY is *present* and just doesn't mature (because it doesn't get to the vacuole). If not for this pulse-chase experiment, one might wonder whether some of the mutants destabilized pCPY rather than affecting its trafficking. In fact, the authors assert that some mutants *do* affect stability, but no one in discussion section seems to think that the blot actually supports this conclusion.
+ A time course experiment is still really useful for deciphering the mechanism.
+ This was a competitive field at the time, so the authors may have been aware that their pulse-chase experiment was less than perfect but felt it was worth going ahead and submitting anyway. 
+ It seems clear that the authors did successfully isolate mutants defective in vacuole targeting. The paper's other shortcomings can be forgiven.


Even in 1986, it was possible to use PCR to see if a gene is in the correct locus - I think you use a primer in the gene and a primer nearby but outside the gene, and you get the right size product only if it is in the right locus.

The strains listed in the Methods are as follows:

| strain name | genotypes | what is it for |
| ---- | ----- | ---- |
| SEY2101 | *MATa* *ura3-52* *leu2-3,-112* *suc2-&Delta;9* *ade2-1* | Base strain lacking functional invertase (*suc2-&Delta;9*) so deficient in growing on sucrose. Possesses endogenous CPY. *ura3-52* and *leu2-3,-112* are probably just selection markers. The *ade2-1* is of unknown significance (also see below). |
| SEY2102 | *MAT&alpha;* *ura3-52* *leu2-3,-112* *suc2-&Delta;9* *his4-519* | Same as SEY2101 but &alpha; instead of a. Except that it has *his4-519* instead of *ade2-1*. Nobody in our section could figure out why this is; possibly these were just the strains they could purchase commercially. |
| SEY2108 | *MAT&alpha;* *ura3-52* *leu2-3,-112* *suc2-&Delta;9* *&Delta;prc1::LEU2+* | SEY2102 with *PRC1* deleted, and *LEU2* introduced in the *PRC1* locus for selection. They re-introduced CPY into this strain for Figures 2-3. |
| SEY2109 | *MATa* *ura3-52* *leu2-3,-112* *suc2-&Delta;9* *&Delta;prc1::LEU2+* | Same as SEY2108 but a instead of &alpha;. They re-introduced CPY into this strain for Figures 2-3. |
| SEY2101.1 | *MATa* *ura3-52* *leu2-3,-112* *suc2-&Delta;9* *ade2-1* *&Delta;pep4::LEU2+* | For Figure 1, to show that the fusion protein behaves the same whether or not PEP4, the CPY peptidase, is present, i.e. PEP4 does not cleave the fusion protein. |

Need to know why they used the media they did.

Methods
+ Lithium acetate permeabilizes the cell wall to allow transformation.
+ The media needs to lack other carbon sources besides sucrose. For pulse-chase it needs to lack other sources of sulfur.
+ Why [bromocresol purple](http://en.wikipedia.org/wiki/Bromocresol_purple)?

### [Katzmann 2001]: Ubiquitin-Dependent Sorting into the Multivesicular Body Pathway Requires the Function of a Conserved Endosomal Protein Sorting Complex, ESCRT-I

#### Background

At the time, it was known that cell surface receptors were sorted for recycling or degradation via the endocytic pathway, but for most proteins no one knew what determined the fate of a given receptor. Degradation of EGFR, however, was known to require a kinase, a cytosolic tail of EGFR, and a ubiquitin ligase. Absent any of these things, it recycled continuously and caused cancer. This was a hint that ubiquitin was important for its degradation.

This paper shows that three proteins &mdash; Vps23, Vps28 and Vps37 &mdash; formed a complex called ESCRT-I (pronounced "escort").

#### Figures

In Figure 1A, they fuse Vps23 (a known protein) to protA (an immunoprecipitable tag) in order to pull it down. They then find that Vps28 and Vps37 co-IP with it. In Figure 1B, they perform "gel filtration analysis" - basically chromatography - and each lane on the gel represents one fraction of eluent coming off the machine. Elution from a column is sort of a random walk, which is why any given molecule will elute over a few minutes and not all at once - hence the complex is seen in multiple fractions (multiple lanes).

In Figure 2, they've fused Vps23 to GFP so that they can look at its localization. They use Snf7 as an endosome marker. In Figure 2B, there is a lot of green, so under the null hypothesis that Vps23 localizes nowhere in particular (say it's cytosolic), of course the authors would have been able to find *two* cells in which red Snf7 localized in a subset of places where green Vps23 was found. Even the two cherry-picked examples here do not imply Vps23 is exclusively endosomal. Therefore this figure is fairly weak evidence for the subcellular localization of Vps23.

Strains lacking *VPS4* are deficient in cleavage of something, so all of the something is localized at the membrane.

Based on some argument about homology, they think that ubiquitin is important. So they use an error-prone PCR mutagenesis screen to look for LoF mutations in Vps23, and they isolate a mutation in Vps23^M85T which is non-functional but associates with the others to form a complex. This mutation is located in a domain predicted to have something to do with ubiquitination, supporting their idea that ubiquitin will somehow prove to be involved.

In Figure 3, they IP with an antibody against CPS, and then probe the blot with antibodies to CPS and to Ub. DOA4's function is to de-ubiquitylate things, so the *&Delta;doa4* strain in lane 2 has CPS that is more highly ubiquitylated.

The mutants they obtain all affect different steps in the sorting pathway:

| mutant | what step it messes up |
| ------ | ----------------- |
| &Delta;gga1 | Golgi &rarr; early endosome |
| &Delta;pep12 | early endosome &rarr; late endosome |
| &Delta;doa4 | late endosome |
| &Delta;vam3 | late endosome &rarr; multi-vesicular body |

In 2001 it was already known where each of these proteins acted in the secretory pathway. Therefore by knocking each of them out they can determine *where* ubiquitination and de-ubiquitination occur.

When they mutate Gga1, they get *no* ubiquitination at all (empty-looking second-to-last lane), implying that ubiqutination happens after the Golgi. When they knock out Pep12, do get ubiquitination, implying ubiquitination has occurred in the early endosome. Doa4 is known to act in the late endosome, and knocking it out causes CPS to stay heavily ubiquitinated. Vam3 is known to manage trafficking from late endosomes to MVBs, so knocking it out gives CPS more time to interact with Doa4, hence it is fully de-ubiquitinated. Double knockout of Vam3 and Doa4 rescues this and allows CPS to be ubiquitinated.

In Figure 4, they mutate residue K8 and show that this affects ubiquitination; mutating K12 doesn't do as much. From this they conclude that K8 is the likely site of ubiquitination.

The theory of the MVB pathway is that invagination creates vesicles inside the late endosomes, and that anything left in the lumen, unbound to these vesicles, will be degraded.

In Figure 5, to see if ubiquitination is *sufficient* to cause cargo to be saved from ending up in the lumen and getting degraded, they express a constitutively ubiquitinated GFP-CPS and show that it ends up in the lumen. If you delete Vps4 this does not occur. 

In Figure 6 they show that Vps23 can bind ubiquitinated proteins and that someplace around residue 85 is important for this.




[Bankaitis 1986]: http://www.ncbi.nlm.nih.gov/pubmed/3538017 "Bankaitis VA, Johnson LM, Emr SD. Isolation of yeast mutants defective in protein targeting to the vacuole. Proc Natl Acad Sci U S A. 1986 Dec;83(23):9075-9. PubMed PMID: 3538017; PubMed Central PMCID: PMC387077."

[Katzmann 2001]: http://www.ncbi.nlm.nih.gov/pubmed/11511343 "Katzmann DJ, Babst M, Emr SD. Ubiquitin-dependent sorting into the multivesicular body pathway requires the function of a conserved endosomal protein sorting complex, ESCRT-I. Cell. 2001 Jul 27;106(2):145-55. PubMed PMID: 11511343."


