---
layout: post
title: "Structural mutants of PrP"
author: ericminikel
date: 2014-04-29 09:32:10
---
<p>Over 30 different structurally mutated versions of PrP have been observed naturally or experimentally. Understanding the behavior of these mutant PrPs can help to teach us what parts of PrP are required for prion replication and for neurotoxicity. The mutants which were studied in mice and in N2a cells up to 2008 were reviewed elegantly in <a href="http://www.ncbi.nlm.nih.gov/pubmed/18558863">Aguzzi 2008</a>. This post is intended to provide an encyclopedic reference to all these mutants, including those observed in humans and other species. Please comment if anything is incorrect or missing. My aim will be to keep this post up to date with all the mutants reported in the literature as time goes on and provide a detailed description of each.</p>
<p>This table summarizes all of the reported structural mutants. Detailed descriptions follow. The columns at right indicate:</p>
<ol>
<li><strong>Produces prions</strong> &#8211; does this PrP sequence, expressed in vivo, spontaneously give rise to prions? For the purposes of this column I&#8217;ll accept amyloid formation as evidence of a prion having been formed &#8211; but see sections below for more details on transmissibility to mice.</li>
<li><strong>Accepts prions</strong> &#8211; does this PrP sequence, expressed in vivo, support prion replication &#8211; i.e. can humans with this allele develop prion disease, or can this allele&#8217;s protein product be converted to PrP<sup>Sc</sup> in the mouse brain? Note that several sequences have been described to <em>not</em> support prion replication in ScN2a cells, but that I do not consider this to evidence against replication competence, since some PrP mutants only support prion disease in vivo after a very long incubation time, at least on first passage, and it is not clear that N2a cells provide the amount of opportunity for prion adaptation that is afforded in the living brain.</li>
<li><strong>Other phenotype</strong> &#8211; does this PrP sequence, expressed in vivo, give rise to any other spontaneous phenotype?</li>
</ol>
<p>I have left these columns blank where the answer has not been characterized.</p>
<p><a href="/wp-content/uploads/2014/04/prp-deletion-mutants.png"><img src="/wp-content/uploads/2014/04/601x1201xprp-deletion-mutants.png"/></a></p>
<p><em><a href="https://github.com/ericminikel/prp-misc/blob/master/prp-struct-mutants.r">Code that created this graphic</a></em></p>
<p><strong>HuPrP Y145X</strong></p>
<p><img src="/wp-content/uploads/2014/04/Y145X.png"/></p>
<p>This truncating mutation has been observed in one patient with cerebral amyloid angiopathy [<a href="http://www.ncbi.nlm.nih.gov/pubmed/8097911">Kitamoto 1993</a>, <a href="http://www.ncbi.nlm.nih.gov/pubmed/8570627/">Ghetti 1996</a>]. It has never been transmitted to mice, but the PrP amyloid plaques in this disease do include the wild-type allele&#8217;s protein product, as evidenced by the immunoreactivity of plaques to antisera against PrP220-231, a region not present in the Y145X allele [<a href="http://www.ncbi.nlm.nih.gov/pubmed/8570627/">Ghetti 1996</a>]. To me this seems to be evidence of intermolecular transmissibility at a minimum, even if interorganismal transmissibility has not been observed. Attempts to model this mutation in the mouse have been unsuccessful (see 144# below).</p>
<p><strong>HuPrP Y163X</strong></p>
<p><img src="/wp-content/uploads/2014/04/Y163X.png"/></p>
<p>Of the nonsense mutations observed in humans, Y163X is the only one that has been extensively characterized and observed segregating with disease in a family [<a href="http://www.ncbi.nlm.nih.gov/pubmed/24224623">Mead 2013</a>]. It leads to eventually to cerebral amyloid angiopathy, but first has a decades-long disease course with peripheral PrP plaque formation, chronic diarrhea and polyneuropathy. Transmission to mice was unsuccessful, but as with Y145X, the plaques reacted with a C-terminal antibody (Pri-917, against amino acids 216â€“221), suggesting that the wild-type allele had been recruited. The Y163X mutation was first reported in [<a href="http://www.ncbi.nlm.nih.gov/pubmed/19225789/">Revesz 2009</a>].</p>
<p><strong>HuPrP D178fs25X</strong></p>
<p><img src="/wp-content/uploads/2014/04/D178fs25X.png"/></p>
<p>This 2-bp frameshift deletion at codon 178 creates a secreted mutant PrP with a cerebral amyloid angiopathy phenotype apparently similar to the nonsense mutations above and below, and diarrhea, perhaps reminiscent of Y163X. This was only characterized in one patient, but family history indicated a parent and grandparent with the same condition [<a href="http://www.ncbi.nlm.nih.gov/pubmed/23577609">Matsuzono 2013</a>]. Transmissibility was not tested.</p>
<p><strong>HuPrP Y226X &#038; Q227X</strong></p>
<p><img src="/wp-content/uploads/2014/04/Q227X.png"/></p>
<p>I&#8217;m lumping these two together since they are at adjacent codons and leave virtually the entire protein intact &#8211; in fact, the stop codons are only one or two residues away from the site where ADAM10 cleaves PrP in its <a href="/2013/06/29/proteolytic-shedding-of-prion-protein/" title="Proteolytic shedding of prion protein">natural proteolytic shedding</a> process. These have each been observed in one patient with cerebral amyloid angiopathy, and transmissibility has not been tested [<a href="http://www.ncbi.nlm.nih.gov/pubmed/19911184/">Jansen 2010</a>]. </p>
<p><strong>Goat 32Stop</strong></p>
<p><img src="/wp-content/uploads/2014/04/G29X.png"/></p>
<p>This naturally occurring nonsense mutation, homologous to G29X in HuPrP numbering, was observed in a Norwegian dairy goat herd [<a href="http://www.ncbi.nlm.nih.gov/pubmed/23249298/">Benestad 2012</a>]. It would leave only a 7 amino acid peptide after signal peptide cleavage, and appears to be effectively a knockout allele. Neither heterozygotes nor homozygotes were reported to have any phenotype.</p>
<p><strong>HuPrP OPRD mutants</strong></p>
<p><img src="/wp-content/uploads/2014/04/2-OPRD.png"/></p>
<p>A single octapeptide repeat deletion (1-OPRD) is a neutral variant found in Mediterranean populations [<a href="http://www.ncbi.nlm.nih.gov/pubmed/8100163">Palmer 1993</a>], with about a ~1% allele frequency. 2-OPRD alleles have apparently been reported in patients with prion disease, though I&#8217;ve only seen this mentioned in figures [e.g. <a href="http://www.ncbi.nlm.nih.gov/pubmed/20583301">Beck 2010</a>] and was unable to find the original citation in a Google Scholar search.</p>
<p><strong>HuPrP OPRI mutants / MoPrP PG14</strong></p>
<p><img src="/wp-content/uploads/2014/04/OPRI.png"/></p>
<p>There is a vast literature on octapeptide repeat insertions (OPRIs) in humans, which I&#8217;m not terribly familiar with and deserves a separate post at some point. OPRIs of varying lengths have been reported. 1-, 2- and 3-OPRI have only been in single case reports and may not be pathogenic, 4-OPRI has been seen more often but may be incompletely penetrant [<a href="http://www.ncbi.nlm.nih.gov/pubmed/21616973">Kaski 2011</a>] while &ge; 5-OPRI can be somewhat more confidently characterized as Mendelian disease alleles [<a href="http://www.ncbi.nlm.nih.gov/pubmed/17709704">Mead 2007</a>, <a href="http://www.ncbi.nlm.nih.gov/pubmed/16923955/">Mead 2006</a>]. The OPRI mutations have a fairly early age of onset (e.g. 35 for 6-OPRI), a long disease course (several years), and are transmissible to HuPrP mice [<a href="http://www.ncbi.nlm.nih.gov/pubmed/16923955/">Mead 2006</a>].</p>
<p>The longest known such mutation is 9-OPRI, also known as PG14 (9 inserted repeats plus 4 endogenous repeats plus 1 nonarepeat = 14, and the repeats are rich in proline and glycine, hence PG14). A mouse model of the PG14 mutation [<a href="http://www.ncbi.nlm.nih.gov/pubmed/9883727">Chiesa 1998</a>] developed spontaneous neurological illness characterized by cerebellar granule cell loss, even with only about 1x expression of the mutant protein. The disease was fastest on a <em>Prnp</em> knockout background but was still present on a wild-type background. The PrP in the brains of these mice exhibited some properties of PrP<sup>Sc</sup> including detergent insolubility and some (limited) degree of protease resistance, but the disease could not be transmitted to wild-type mice [<a href="http://www.ncbi.nlm.nih.gov/pubmed/12805461">Chiesa 2003</a>].</p>
<p><strong>MoPrP &Delta;Nterm / &Delta;A / &Delta;32-80</strong></p>
<p><img src="/wp-content/uploads/2014/04/PrPdNterm1.png"/></p>
<p>Known by various names, this was the first mutant expressed in mice [<a href="http://www.ncbi.nlm.nih.gov/pubmed/8635458/">Fischer 1996</a>]. Because PrP<sup>Sc</sup> can have its N terminus digested by proteinase K or removed by beta cleavage and yet retain its infectivity, this experiment was designed to test the hypothesis that PrP without the N terminus could still support prion disease <em>in vivo</em>. The extreme N-terminal tip of the finished peptide, amino acids 23-31 &mdash; KKRPKPGGW &mdash; were left in simply because the authors weren&#8217;t sure whether PrP would be properly directed to the ER and have its signal peptide cleaved without them. It turns out (see &Delta;23-88 below) that PrP does localize properly without those nine amino acids. The &Delta;Nterm mice had no spontaneous phenotype, but could be infected with RML prions and had pathology and incubation times similar to mice expressing wild-type PrP.</p>
<p><strong>MoPrP &Delta;Nco / &Delta;B / &Delta;69-84</strong></p>
<p><img src="/wp-content/uploads/2014/04/PrPdNco1.png"/></p>
<p>Also reported in [<a href="http://www.ncbi.nlm.nih.gov/pubmed/8635458/">Fischer 1996</a>]. Similar to &Delta;Nterm, this mutant had no spontaneous phenotype but seemed to support prion disease upon inoculation just like wild-type PrP.</p>
<p><strong>MoPrP &Delta;C / &Delta;32-93</strong></p>
<p><img src="/wp-content/uploads/2014/04/PrPdelC.png"/></p>
<p>These mice were characterized in <a href="http://www.ncbi.nlm.nih.gov/pubmed/9568713">Shmerling 1998</a> as having no spontaneous phenotype, and their susceptibility to prions was examined in <a href="http://www.ncbi.nlm.nih.gov/pubmed/10985358">Flechsig 2000</a>. The mice were susceptible to RML prions, but only weakly so. Their incubation times were longer than wild-type mice, even at high levels of transgene expression (e.g. ~260 days to terminal illness at a 4x expression level), they produced prion infectivity titers that were 10-30x lower than wild-type mice and NaPTA-precipitable PrP<sup>Sc</sup> levels that were 50x lower, and they exhibited neuropathology only in the spinal cord. This reduced susceptibility to RML didn&#8217;t appear to be due to a transmission barrier, because incubation times were not reduced on second passage. There was some ambiguity here, though &#8211; the second passage mice did have some (albeit limited) pathology in the brain, unlike the first passage mice.</p>
<p><strong>MoPrP &Delta;D / &Delta;32-106</strong></p>
<p><img src="/wp-content/uploads/2014/04/PrPdelD.png"/></p>
<p>These mice were characterized in <a href="http://www.ncbi.nlm.nih.gov/pubmed/9568713">Shmerling 1998</a> as having no spontaneous phenotype, but their susceptibility to prions was never formally reported. However, both <a href="http://www.ncbi.nlm.nih.gov/pubmed/14522848">Weissman &#038; Flechsig 2003</a> and <a href="http://www.ncbi.nlm.nih.gov/pubmed/18558863">Aguzzi 2008</a> cite &#8220;unpublished data&#8221; to state that these mice were not susceptible to prions.</p>
<p><strong>MoPrP &Delta;E / &Delta;32-121</strong></p>
<p><img src="/wp-content/uploads/2014/04/PrPdelE.png"/></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/9568713">Shmerling 1998</a> reported that these mice have a spontaneous fatal disease involving neuronal loss in the cerebellum. The disease is most pronounced when the mutant allele is expressed on a PrP knockout background; it can be dose-dependently rescued by wild-type PrP. They were unable to transmit this disease to Tga20 mice, which overexpress wild-type PrP. </p>
<p><strong>MoPrP &Delta;F / &Delta;32-134</strong></p>
<p><img src="/wp-content/uploads/2014/04/PrPdelF.png"/></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/9568713">Shmerling 1998</a> reported that these mice the same spontaneous cerebellar disease as &Delta;23-121 mice, but with a slightly later onset. As with &Delta;23-121, disease is most pronounced when the mutant allele is expressed on a PrP knockout background; it can be dose-dependently rescued (i.e. onset delayed) by wild-type PrP. They were unable to transmit this disease to Tga20 mice, which overexpress wild-type PrP.</p>
<p>The Harris lab also now uses the &Delta;23-134 construct and has shown that, like &Delta;CR (below), it makes cells hypersensitive to cationic antibotics [<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2844219/">Massignan 2010</a>].</p>
<p><strong>MoPrP &Delta;CD / &Delta;94-134</strong></p>
<p><img src="/wp-content/uploads/2014/04/PrPdelCD.png"/></p>
<p>Looking at the phenotypes of the A &#8211; F mice, Aguzzi observed that it was only the deletions (E and F) that extended into the &#8220;Central Domain&#8221; (CD) of the protein that caused spontaneous disease. The &Delta;CD mice were therefore created to test the hypothesis that only the central domain had to be deleted in order to cause spontaneous cerebellar disease. Sure enough, the &Delta;CD mice got a disease like that of &Delta;E and &Delta;F mice, but with an even earlier onset and a requirement of more wild-type PrP to rescue it [<a href="http://www.ncbi.nlm.nih.gov/pubmed/17245436">Baumann 2007</a>]. </p>
<p><strong>MoPrP &Delta;pHC / &Delta;121-134</strong></p>
<p><img src="/wp-content/uploads/2014/04/PrPdelpHC.png"/></p>
<p>The &#8220;<u>del</u>etion of <u>p</u>art of the <u>H</u>ydrophobic <u>C</u>ore&#8221; or &Delta;pHC mice were created along with the &Delta;CD mice to see whether a smaller deletion would cause the same phenotype [<a href="http://www.ncbi.nlm.nih.gov/pubmed/17245436">Baumann 2007</a>]. The &Delta;pHC mice had no spontaneous phenotype, yet the mutant did behave differently than wild-type PrP. PrP&Delta;pHC was less good than wild-type PrP at rescuing the &Delta;F phenotype, giving only a marginal (p = .06) delay in disease onset, and it actually made the &Delta;CD disease even <em>more</em> rapid. The authors concluded that &Delta;pHC is a &#8220;partial agonist.&#8221; See more discussion at the bottom of this post.</p>
<p><strong>MoPrP &Delta;CR / &Delta;105-125</strong></p>
<p><img src="/wp-content/uploads/2014/04/PrPdCR.png"/></p>
<p>This creation of the Harris lab is even more lethal than &Delta;CD, which was the most lethal of the Zurich series of mutants. &Delta;105-125, in which the highly conserved <u>C</u>entral <u>R</u>egion is deleted, makes cells hypersensitive to cationic antibotics such as zeocin [<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2844219/">Massignan 2010</a>], causes spontaneous transmembrane currents [<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2924115/">Solomon 2010</a>], and, when expressed on a knockout background, causes lethal cerebellar granule cell loss within 1 week after birth, even at 1x expression levels or less [<a href="http://www.ncbi.nlm.nih.gov/pubmed/17245437/">Li 2007</a>]. Co-expression of wild-type PrP can rescue this phenotype, though it takes 5x wild-type PrP to extend life to about 1 year. The phenotype of these mice seems almost identical to that of &Delta;CD mice; the age of onset is, if anything, even earlier and the amount of wild-type PrP required for rescue even greater.</p>
<p><strong>C1 / MoPrP &Delta;23-111</strong></p>
<p><img src="/wp-content/uploads/2014/04/C1.png"/></p>
<p>As introduced <a href="/2013/06/26/proteolysis-of-prion-protein-in-the-central-region-alpha-and-beta-cleavage/" title="Proteolysis of prion protein in the central region: alpha and beta cleavage">here</a>, PrP undergoes a proteolytic event called alpha cleavage which produces the N1 and C1 fragments. It has recently been shown that there are at least three different alpha cleavage sites [<a href="http://www.ncbi.nlm.nih.gov/pubmed/24247244">McDonald 2014</a>], but the first identified site was on either side of H111 (human sequence) / H110 (mouse sequence) [<a href="http://www.ncbi.nlm.nih.gov/pubmed/8093841">Harris 1993</a>]. The Harris lab therefore created a mouse which constitutively expresses only the C1 fragment &#8211; in fact, they deleted up through 111 (mouse numbering), leaving a protein which begins with the AGAAAAGA palindromic region [<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3243553/">Westergard 2011</a>]. The mice had no spontaneous phenotype, and could not be infected with prions. In fact, co-expression of C1 with wild-type PrP extended the incubation time for RML prions, implying that the C1 fragment is actually dominant negative against prion propagation.</p>
<p><strong>MoPrP &Delta;23-134</strong></p>
<p><img src="/wp-content/uploads/2014/04/PrPd23-134.png"/></p>
<p>These mice were also created by the Harris lab and, like the C1 mice, don&#8217;t develop any spontaneous phenotype (even with higher expression levels than the &Delta;32-134 mice above, which Harris used as foils to these mice) and are not susceptible to prion infection [<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3243553/">Westergard 2011</a>]. Like C1, this fragment appears to be dominant negative against scrapie infection &#8211; when co-expressed with wild-type PrP, it prolongs the incubation period, though the effect is smaller than with C1.</p>
<p><strong>MHM2 and MoPrP &Delta;23-88</strong></p>
<p><img src="/wp-content/uploads/2014/04/PrPd23-88.png"/></p>
<p>MHM2 is a chimeric molecule composed of mouse PrP with 2 hamsterized amino acids (109 and 112 in hamster/human numbering) completing the 3F4 epitope (KTNMKHM, aa 106-112 in hamster/human numbering). The MHM2 &Delta;23-88 mutant additionally has much of the N terminus lopped off, including the N-terminal tip. The Prusiner lab originally expressed this mutant in N2a cells and showed that it was capable of converting to PK-resistant PrP<sup>Sc</sup> [<a href="http://www.ncbi.nlm.nih.gov/pubmed/8475059/">Rogers 1993</a>], then expressed it in mice and showed that it caused no spontaneous phenotype [<a href="http://www.ncbi.nlm.nih.gov/pubmed/9212101">Muramoto 1997</a>]. The next question was whether these mice were susceptible to prion disease, and the answer turned out to be complicated. Mice expressing only MHM2 &Delta;23-88 could not be infected with RML prions at all [<a href="http://www.ncbi.nlm.nih.gov/pubmed/10102274">Supattapone 1999</a>]. Co-expression of one allele of wild-type MoPrP, however, rendered these mice susceptible [<a href="http://www.ncbi.nlm.nih.gov/pubmed/10102274">Supattapone 1999</a>]. It wasn&#8217;t just that only the wild-type allele was supporting the prion replication, however, because (1) the incubation time was 257 days, compared to &gt; 400 days for mice expressing <em>only</em> one wild-type allele, and moreover, (2) some of the PK-resistant PrP<sup>Sc</sup> reacted with the 3F4 antibody, which recognizes MHM2 &Delta;23-88 and not wild-type MoPrP.</p>
<p>A couple years later, they did a more detailed characterization to unravel why the MHM2 &Delta;23-88 sequence was so resistant to mouse prions [<a href="http://www.ncbi.nlm.nih.gov/pubmed/11152514/">Supattapone 2001</a>]. In this study, they compared mice with and without the 3F4 epitope and the 23-88 deletion: MoPrP, MHM2, MoPrP &Delta;23-88, and MHM2 &Delta;23-88. They found that no matter what they did, they simply could not transmit prions to mice that only expressed MHM2 &Delta;23-88. Yet surprisingly, mice expressing only MoPrP &Delta;23-88 <em>were</em> susceptible to RML prions, as were mice that expressed MHM2 without the deletion. The eventual disease phenotype and neuropathology of the MoPrP &Delta;23-88 mice were described as representing &#8220;typical signs of scrapie&#8221;, yet the incubation time was surprisingly long: 160 days on first passage and 140 days on second passage, compared to ~50 days in Tg4053 mice overexpressing wild-type MoPrP, even though the MoPrP &Delta;23-88 mice expressed twice as much PrP as Tg4053 mice did. Mice expressing full-length MHM2 PrP had similarly extended incubation times. Therefore the authors concluded that the resistance of MHM2 &Delta;23-88 mice to prion infection was the result of an interaction between the fundamentally lower susceptibility of MHM2 PrP to prions and the fundamentally lower susceptibility of &Delta;23-88 PrP to prions.</p>
<p>The crucial piece of this experiment that is missing for me is that they never tested the prion infectivity titers in the brains of the MoPrP &Delta;23-88 mice at the time of terminal disease. Hypothesis: if residues 23-88 are involved in a crucial neurotoxic pathway, as much evidence supports [<a href="http://www.ncbi.nlm.nih.gov/pubmed/23903654">Sonati 2013</a>], then the long incubation times might be due to reduced neurotoxicity, not to reduced prion replication. If so, then by the time they finally do succumb to prion disease, they should have had time to accumulate much greater infectivity in their brains than mice expressing wild-type PrP.</p>
<p><strong>MHM2 &Delta;23-88 &Delta;95-107 / &Delta;STE</strong></p>
<p><img src="/wp-content/uploads/2014/04/PrPd23-88d95-107.png"/></p>
<p>This and the next several mutants were created by the Prusiner lab in a systematic campaign to figure out which regions of PrP secondary structure were required for prion replication. The 95-107 segment (96-108 in human numbering) deleted in this particular mutant was considered to be the &#8220;stop transfer effector&#8221; (STE) because its series of positive charges (HSQWNKPSKPKTN in HuPrP) was predicted to help direct insertion of the hydrophobic core into the membrane in transmembrane forms of PrP &#8211; see <a href="/2013/02/24/cell-biology-04-the-secretory-pathway/">this post</a> for some cell biology basics.</p>
<p>In ScN2a cells, MHM2 &Delta;23-88 supported prion replication (though in the presence of wild-type PrP) [<a href="http://www.ncbi.nlm.nih.gov/pubmed/8475059/">Rogers 1993</a>], but additionally deleting 95-107 rendered the molecule unable to convert to PrP<sup>Sc</sup> in ScN2a cells [<a href="http://www.ncbi.nlm.nih.gov/pubmed/8986833/">Muramoto 1996</a>]. When expressed in transgenic mice on a knockout background, MHM2 &Delta;23-88 &Delta;95-107 caused no spontaneous phenotype [<a href="http://www.ncbi.nlm.nih.gov/pubmed/9212101">Muramoto 1997</a>]. The susceptibility of these mice to prions was never tested.</p>
<p><strong>MHM2 &Delta;23-88 &Delta;108-121</strong></p>
<p><img src="/wp-content/uploads/2014/04/PrPd23-88d108-121.png"/></p>
<p>When the Prusiner lab went on its campaign of deleting PrP secondary structure regions, it used a computational prediction of the structure of PrP<sup>C</sup> which later turned out not to be entirely accurate. MoPrP residues 108-121 (109-122 in human numbering), deleted in this mutant, were predicted to be an alpha helix dubbed H1, but the first NMR structure of the globular domain of PrP<sup>C</sup>, published the same year [<a href="http://www.ncbi.nlm.nih.gov/pubmed/8700211">Riek 1996</a>], showed that there was in fact no alpha helix here. In any event, this mutant did not support prion replication in ScN2a cells [<a href="http://www.ncbi.nlm.nih.gov/pubmed/8986833/">Muramoto 1996</a>] and caused no phenotype in transgenic mice [<a href="http://www.ncbi.nlm.nih.gov/pubmed/9212101">Muramoto 1997</a>]. The susceptibility of the mice to prions was never studied.</p>
<p><strong>MHM2 &Delta;23-88 &Delta;122-140</strong></p>
<p><img src="/wp-content/uploads/2014/04/PrPd23-88d122-140.png"/></p>
<p>This mutant did not support prion replication in ScN2a cells [<a href="http://www.ncbi.nlm.nih.gov/pubmed/8986833/">Muramoto 1996</a>]. Alone among the Prusiner series of MHM2 mutants, this one was never successfully expressed in transgenic mice. It is not known whether this is due to bad luck, toxicity or something else.</p>
<p><strong>MHM2 &Delta;23-88 &Delta;141-221</strong></p>
<p><img src="/wp-content/uploads/2014/04/PrPd23-88d141-221.png"/></p>
<p>Possibly apocryphal. <a href="http://www.ncbi.nlm.nih.gov/pubmed/18558863">Aguzzi 2008</a> refers to such a mutant, citing [<a href="http://www.ncbi.nlm.nih.gov/pubmed/9212101">Muramoto 1997</a>], yet as far as I can tell this mutant is not present in the Muramoto paper.</p>
<p><strong>MHM2 &Delta;23-88 &Delta;141-176 / PrP106 / &#8220;miniprions&#8221;</strong></p>
<p><img src="/wp-content/uploads/2014/04/PrP106.png"/></p>
<p>Alone among the Prusiner series of mutants, MHM2 &Delta;23-88 &Delta;141-176 supported prion replication in cell culture [<a href="http://www.ncbi.nlm.nih.gov/pubmed/8986833/">Muramoto 1996</a>]. Residues 141-176 (human numbering 142-177) comprise alpha helix 1, beta sheet 2, and the loop between beta sheet 2 and alpha helix 2. Because these deletions leave only a 106-residue, GPI-anchored peptide, the protein was dubbed PrP106. There was no spontaneous phenotype in vivo [<a href="http://www.ncbi.nlm.nih.gov/pubmed/9212101">Muramoto 1997</a>].</p>
<p>The susceptibility of these mice to prion infection was extensively studied [<a href="http://www.ncbi.nlm.nih.gov/pubmed/10102274">Supattapone 1999</a>] and was even referenced in Prusiner&#8217;s Nobel lecture [<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC33918/">Prusiner 1998</a>]. Tg(PrP106) mice were susceptible to RML prions in 300 days on first passage, and 66 days on second passage. It was proposed that the &#8220;miniprions&#8221; created in these mice might be useful for structural studies of PrP, but if these studies were done I never saw the paper on them. The neuropathology in the mice was described as &#8220;characteristic of experimental scrapie&#8221;.  </p>
<p><strong>MHM2 &Delta;23-88 &Delta;177-200</strong></p>
<p><img src="/wp-content/uploads/2014/04/PrPd23-88d177-200.png"/></p>
<p>According to <a href="http://www.ncbi.nlm.nih.gov/pubmed/8700211">Riek 1996</a>, &alpha;-helix 2 encompasses amino acids 179-193 (mouse numbering). This Prusiner lab deletion mutant therefore includes all of &#038;alpha-helix 2 and a bit extra on both sides, though in the papers [<a href="http://www.ncbi.nlm.nih.gov/pubmed/8986833/">Muramoto 1996</a>, <a href="http://www.ncbi.nlm.nih.gov/pubmed/9212101">Muramoto 1997</a>] it is incorrectly referred to as H3. This molecule couldn&#8217;t be converted to PrP<sup>Sc</sup> in ScN2a cells, so the susceptibility of mice was never tested. The male mice expressing this mutant did, however, suffer a fatal neuronal storage disease, with cytoplasmic neuronal inclusions of insoluble mutant PrP [<a href="http://www.ncbi.nlm.nih.gov/pubmed/9212101">Muramoto 1997</a>]. The phenotype looked quite different from that of scrapie infection, and the transmissibility of this disease was never tested.</p>
<p><strong>MHM2 &Delta;23-88 &Delta;201-217</strong></p>
<p><img src="/wp-content/uploads/2014/04/PrPd23-88d201-217.png"/></p>
<p>Similar to the above, this was the Prusiner lab&#8217;s deletion of &alpha;-helix 3 (which spans 201-218 [<a href="http://www.ncbi.nlm.nih.gov/pubmed/8700211">Riek 1996</a>]), which they originally called H4. Like &Delta;23-88 &Delta;177-200, this mutant did not convert to PrP<sup>Sc</sup> in ScN2a cells [<a href="http://www.ncbi.nlm.nih.gov/pubmed/8986833/">Muramoto 1996</a>], but caused a spontaneous neuronal storage disease in mice [<a href="http://www.ncbi.nlm.nih.gov/pubmed/9212101">Muramoto 1997</a>]. This deletion was more fully penetrant than the &Delta;177-200 deletion and affected both male and female mice.</p>
<p><strong>MHM2 &Delta;23-88 C178A</strong></p>
<p><img src="/wp-content/uploads/2014/04/PrPd23-88C178A.png"/></p>
<p>Though C178A is a point mutation, this is included in the list of structural mutants because it abolishes PrP&#8217;s disulfide bond. This mutant did not convert to PrP<sup>Sc</sup> in ScN2a cells [<a href="http://www.ncbi.nlm.nih.gov/pubmed/8986833/">Muramoto 1996</a>] and did not cause any phenotype in mice [<a href="http://www.ncbi.nlm.nih.gov/pubmed/9212101">Muramoto 1997</a>]. The susceptibility of the mice to prion infection was never tested.</p>
<p><strong>MHM2 144#</strong></p>
<p><img src="/wp-content/uploads/2014/04/PrP144X.png"/></p>
<p>This (along with the next entry) was the Prusiner lab&#8217;s attempt to create a mouse model of the Y145X mutation seen in humans (see above) and, simultaneously, to determine whether deleting both &alpha;-helices 2 and 3 would abrogate the neuronal storage disease phenotype. However, they never obtained cells nor mice with detectable expression of this mutant [<a href="http://www.ncbi.nlm.nih.gov/pubmed/9212101">Muramoto 1997</a>].</p>
<p><strong>MHM2 &Delta;23-88 144#</strong></p>
<p><img src="/wp-content/uploads/2014/04/PrPd23-88-144X.png"/></p>
<p>See MHM2 144# above. The Prusiner lab never obtained cells nor mice with detectable expression of this mutant [<a href="http://www.ncbi.nlm.nih.gov/pubmed/9212101">Muramoto 1997</a>].</p>
<p><strong>MHM2 &Delta;144-227</strong></p>
<p><img src="/wp-content/uploads/2014/04/PrPd144-227.png"/></p>
<p>This mutant would have expressed the equivalent of the Y145X mutant but GPI-anchored instead of secreted. However, the Prusiner lab never obtained cells nor mice with detectable expression of this mutant [<a href="http://www.ncbi.nlm.nih.gov/pubmed/9212101">Muramoto 1997</a>].</p>
<p><strong>cyPrP</strong></p>
<p><img src="/wp-content/uploads/2014/04/cyPrP.png"/></p>
<p>Ma Jiyan and Susan Lindquist deleted both the signal peptide (which directs PrP to the ER) and the GPI signal in order to create what they dubbed cyPrP &#8211; a version of MoPrP that is translated directly into the cytosol [<a href="http://www.ncbi.nlm.nih.gov/pubmed/12386337">Ma 2002a</a> (<a href="http://lindquistlab.wi.mit.edu/PDFs/Ma2002Science.pdf">ft</a>)]. cyPrP was cytotoxic in neuroblastoma cells, but they did manage to create mice expressing the mutant, which became sick around 30 days of age and died around 70 days of age, due to atrophy of cerebellar granule cells. The original paper tied this observation into the Lindquist lab&#8217;s larger story about prion neurotoxicity being caused by <a href="/2013/07/25/the-role-of-the-proteasome-in-prp-degradation-and-prion-disease/" title="The role of the proteasome in PrP degradation and prion disease">blockade of the proteasome</a>, and indeed, it was published back-to-back with another paper <a href="http://www.ncbi.nlm.nih.gov/pubmed/12386336">Ma 2002b</a> (<a href="http://lindquistlab.wi.mit.edu/wp-content/uploads/2013/06/Ma2002Science2.pdf">ft</a>)] about how cytosolic accumulation of PrP due to proteasome inhibition leads to the rise of a PrP<sup>Sc</sup>-like conformation. Ma has continued to study cyPrP since studying his own lab and now believes that the neurotoxicity of cyPrP requires that the hydrophobic core of PrP (amino acids ~112-134) be able to associate with the ER membrane [<a href="http://www.ncbi.nlm.nih.gov/pubmed/16537534">Wang 2006</a>, <a href="http://www.ncbi.nlm.nih.gov/pubmed/19281844/">Wang 2009</a>].</p>
<p><strong>MoPrP &Delta;GPI</strong></p>
<p><img src="/wp-content/uploads/2014/04/delGPI.png"/></p>
<p>A few different lines of mice have been created which express full-length PrP which lacks the GPI anchor and is thus secreted. The earliest reports had to be revised a few times [<a href="http://www.ncbi.nlm.nih.gov/pubmed/15933194/">Chesebro 2005</a>, <a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2566293/">Trifilo 2008</a>, <a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2832701/">Chesebro 2010</a>]. Secreted PrP appears to be preferentially degraded, making it hard to achieve an expression level comparable to wild-type PrP. At low expression levels, it is non-pathogenic, but when PrP &Delta;GPI is expressed in mice at &gt; 1x levels, it spontaneously produces a lethal prion infection transmissible to wild-type mice [<a href="http://www.ncbi.nlm.nih.gov/pubmed/22160704">Stohr 2011</a>]. The mice can also be infected with prions. Co-expression with wild-type PrP hastens the spontaneous disease, but co-expression with &Delta;23-88 does not.</p>
