---
layout: post
title:  "Genetics in medicine 8: inflammatory bowel disease"
date:   2015-03-27 12:01:00
author: ericminikel
tag: genetics-228
location: Cambridge, MA
summary200: ""
---

*These are my notes from week 8 of Harvard's [Genetics 228: Genetics in Medicine from Bench to Bedside](http://www2.massgeneral.org/bbs/gen228/gen228_syllabus_material.asp) course, held on March 27, 2015.*

### Background

For journal club today, my group is presenting the paper *Genetic and epigenetic fine mapping of causal autoimmune disease variants* [[Farh & Marson 2015]]. Here's a quick backgrounder.

#### Successes and limitations of genome-wide association studies

The concept of a genome-wide association study (GWAS) is introduced [here](/2014/11/21/genetics-25/). The first GWAS is sometimes credited to be a study of 116,000 SNPs in 96 cases and 50 controls for age-related macular degeneration in 2005 [[Klein 2005]]. But it was not until a couple of years later that GWAS really hit their stride, reaching the scale needed for locus discovery in common diseases, when the Wellcome Trust Case Control Consortium genotyped 17,000 people on 500,000 SNPs [[WTCCC 2007]; see also historical review in [Visscher 2012]]. Since then, over 1,700 GWAS have been published, reporting associations for over 11,000 SNPs [[Welter 2014]].

A few general trends have emerged. Only about 11% of GWAS loci are in protein-coding regions [[Gusev 2014]], which is a dramatic enrichment (as only about 1% of the genome is protein-coding), but still means that the vast majority of causal variants must be non-coding. 57% of GWAS hits lie in open chromatin (DNAse I hypersensitivity regions) [[Gusev 2014]], and expression quantatitive trait loci (eQTLs; variants that affect gene expression) are enriched among GWAS hits [[Nicolae 2010]]. Thus, much of the functional impact of causal variants might lie in their effects on gene expression.

Still, there are only rare cases where people have been able to confidently identify a single variant as driving the GWAS association at a particular locus. The reason that GWAS work at all is also their greatest weakness. Recombination hotspots effectively divide the human genome into haplotype blocks of 2-200kb [[International HapMap Consortium 2003] and also see [here](/2014/11/19/genetics-24/)], within which genetic variants almost always segregate together. Suppose a particular 20kb haplotype block contains 20 SNPs, one of which is causally associated with your phenotype of interest. Then genotyping *any* SNP in that block (or even nearby) will reveal that association. That's why GWAS work - you don't need to genotype the causal SNP to discover an association. But this also makes it incredibly difficult to tell *which* of those 20 SNPs is causal. And until you know the causal variant, it can be hard to reach the sorts of biological insights that will lead to therapeutic avenues or fundamental basic science insights.

Therefore people have devised a lot of strategies for trying to at least narrow down the set of causal variants at a phenotype-associated locus. One approach relevant to this paper is **fine-mapping**. This is when you re-genotype the same people (or more people, if you have them!) with a chip that has a much denser array of probes at the causal locus. For instance, the [ExomeChip](http://genome.sph.umich.edu/wiki/Exome_Chip_Design), [Immunochip](http://www.ibdgenetics.org/projects.html) or [PsychChip](http://www.med.unc.edu/pgc/psychchip). Why didn't we just put all of those SNPs on the GWAS array in the first place, you ask? Capacity on chips isn't free or infinite, and GWAS discovery chips tend to prioritize uniform coverage of the genome and very high quality, readily genotyped SNPs. These fine-mapping chips instead try to exhaustively capture variants found in sequencing studies such as 1000 Genomes or ESP, even if it means some of the probes are of lower quality and do not uniformly cover the genome. Even with fine-mapping, you'll still end up with a lot of variants at one locus all associated with your phenotype, so this is by no means the end of the story. If you can't afford to re-genotype your study participants or don't have any DNA left, you can try to *guess* what other SNPs they have in between SNPs you've genotyped, based on the haplotype structure of 1000 Genomes participants - this is called **imputation**. This is a substitute for fine-mapping and has all the same limitations, plus its own limitations - you won't guess genotypes correctly 100% of the time.

So, as a very simplified example to show how these things work, consider this imaginary locus (code to generate this plot is [here](https://github.com/ericminikel/cureffi/blob/gh-pages/media/2015/03/gwas_visual_aids.r)):

![](/media/2015/03/gwas_visual_aid.png)

You're interested in figuring out, of all the genetic variants in the population (gray bars), which are associated with your phenotype. As a first pass, you genotype a bunch of cases and controls for only a few, relatively uniformly distributed, well-genotyped SNPs (black bars). You find a few SNPs nominally associated, and two SNPs that are significantly associated with your phenotype - they're above the genome-wide significance threshold of p = 5&times;10<sup>-8</sup>. So then you go back with a fine-mapping chip, and you genotype much more (but still not all) of the variation here (orange bars), and you find a bunch more SNPs that are associated, including several that are above the genome-wide threshold and a few that are more significant than your original GWAS hits. But you still don't know which, if any, of the SNPs you've genotyped is causal, so how can you make any biological inferences? That question is where this paper picks up.

#### The genetics of autoimmunity

Of all the diseases and phenotypes that have been subjected to GWAS, autoimmune diseases must rank among either the most challenging or the most successful, depending on your viewpoint. I haven't found an updated figure for 2015, but as of three years ago, there were already 193 independent genomic loci associated with inflammatory bowel disease (IBD) [[Jostins 2012]]. This degree of polygenicity is so vexing that it has motivated whole new statistical approaches to determine whether the associations are largely real or result from confounding effects of population stratification [[Bulik-Sullivan 2015]]. Specific functional variants have been identified in a handful of loci - see for instance *CARD9* [[Rivas 2011]] - but for many or most of these loci, we don't know how the genetic variation impacts gene function, and we don't know if increased or decreased gene expression would associate with disease risk.



[Hindorff 2009]: http://www.ncbi.nlm.nih.gov/pubmed/19474294 "Hindorff LA, Sethupathy P, Junkins HA, Ramos EM, Mehta JP, Collins FS, Manolio TA. Potential etiologic and functional implications of genome-wide association loci for human diseases and traits. Proc Natl Acad Sci U S A. 2009 Jun 9;106(23):9362-7. doi: 10.1073/pnas.0903103106. Epub 2009 May 27. PubMed PMID: 19474294; PubMed Central PMCID: PMC2687147."

[Gusev 2014]: http://www.ncbi.nlm.nih.gov/pubmed/25439723 "Gusev A, Lee SH, Trynka G, Finucane H, Vilhjálmsson BJ, Xu H, Zang C, Ripke S, Bulik-Sullivan B, Stahl E; Schizophrenia Working Group of the Psychiatric Genomics Consortium; SWE-SCZ Consortium, Kähler AK, Hultman CM, Purcell SM, McCarroll SA, Daly M, Pasaniuc B, Sullivan PF, Neale BM, Wray NR, Raychaudhuri S, Price AL; Schizophrenia Working Group of the Psychiatric Genomics Consortium; SWE-SCZ Consortium. Partitioning heritability of regulatory and cell-type-specific variants across 11 common diseases. Am J Hum Genet. 2014 Nov 6;95(5):535-52. doi: 10.1016/j.ajhg.2014.10.004. Epub 2014 Nov 6. PubMed PMID: 25439723; PubMed Central PMCID: PMC4225595."

[Trynka 2011]: http://www.ncbi.nlm.nih.gov/pubmed/22057235 "Trynka G, Hunt KA, Bockett NA, Romanos J, Mistry V, Szperl A, Bakker SF, Bardella MT, Bhaw-Rosun L, Castillejo G, de la Concha EG, de Almeida RC, Dias KR, van Diemen CC, Dubois PC, Duerr RH, Edkins S, Franke L, Fransen K, Gutierrez J, Heap GA, Hrdlickova B, Hunt S, Plaza Izurieta L, Izzo V, Joosten LA, Langford C,  Mazzilli MC, Mein CA, Midah V, Mitrovic M, Mora B, Morelli M, Nutland S, Núñez C, Onengut-Gumuscu S, Pearce K, Platteel M, Polanco I, Potter S, Ribes-Koninckx C, Ricaño-Ponce I, Rich SS, Rybak A, Santiago JL, Senapati S, Sood A, Szajewska H, Troncone R, Varadé J, Wallace C, Wolters VM, Zhernakova A; Spanish Consortium on  the Genetics of Coeliac Disease (CEGEC); PreventCD Study Group; Wellcome Trust Case Control Consortium (WTCCC), Thelma BK, Cukrowska B, Urcelay E, Bilbao JR, Mearin ML, Barisani D, Barrett JC, Plagnol V, Deloukas P, Wijmenga C, van Heel DA. Dense genotyping identifies and localizes multiple common and rare variant association signals in celiac disease. Nat Genet. 2011 Nov 6;43(12):1193-201. doi: 10.1038/ng.998. PubMed PMID: 22057235; PubMed Central PMCID: PMC3242065."

[Cortes 2011]: http://www.ncbi.nlm.nih.gov/pubmed/21345260 "Cortes A, Brown MA. Promise and pitfalls of the Immunochip. Arthritis Res Ther. 2011 Feb 1;13(1):101. doi: 10.1186/ar3204. Review. PubMed PMID: 21345260; PubMed Central PMCID: PMC3157635."

[Parkes 2013]: http://www.ncbi.nlm.nih.gov/pubmed/23917628 "Parkes M, Cortes A, van Heel DA, Brown MA. Genetic insights into common pathways and complex relationships among immune-mediated diseases. Nat Rev Genet. 2013 Sep;14(9):661-73. doi: 10.1038/nrg3502. Epub 2013 Aug 6. Review. PubMed PMID: 23917628."

[Farh & Marson 2015]: http://www.ncbi.nlm.nih.gov/pubmed/25363779 "Farh KK, Marson A, Zhu J, Kleinewietfeld M, Housley WJ, Beik S, Shoresh N, Whitton H, Ryan RJ, Shishkin AA, Hatan M, Carrasco-Alfonso MJ, Mayer D, Luckey CJ, Patsopoulos NA, De Jager PL, Kuchroo VK, Epstein CB, Daly MJ, Hafler DA, Bernstein BE. Genetic and epigenetic fine mapping of causal autoimmune disease variants. Nature. 2015 Feb 19;518(7539):337-43. doi: 10.1038/nature13835. Epub 2014 Oct 29. PubMed PMID: 25363779; PubMed Central PMCID: PMC4336207."

[Klein 2005]: http://www.ncbi.nlm.nih.gov/pubmed/15761122/ "Klein RJ, Zeiss C, Chew EY, Tsai JY, Sackler RS, Haynes C, Henning AK, SanGiovanni JP, Mane SM, Mayne ST, Bracken MB, Ferris FL, Ott J, Barnstable C, Hoh J. Complement factor H polymorphism in age-related macular degeneration. Science. 2005 Apr 15;308(5720):385-9. Epub 2005 Mar 10. PubMed PMID: 15761122; PubMed Central PMCID: PMC1512523."

[WTCCC 2007]: http://www.ncbi.nlm.nih.gov/pubmed/17554300/ "Wellcome Trust Case Control Consortium. Genome-wide association study of 14,000 cases of seven common diseases and 3,000 shared controls. Nature. 2007 Jun 7;447(7145):661-78. PubMed PMID: 17554300; PubMed Central PMCID: PMC2719288."

[Visscher 2012]: http://www.ncbi.nlm.nih.gov/pubmed/22243964/ "Visscher PM, Brown MA, McCarthy MI, Yang J. Five years of GWAS discovery. Am J Hum Genet. 2012 Jan 13;90(1):7-24. doi: 10.1016/j.ajhg.2011.11.029. Review. PubMed PMID: 22243964; PubMed Central PMCID: PMC3257326."

[Welter 2014]: http://www.ncbi.nlm.nih.gov/pubmed/24316577 "Welter D, MacArthur J, Morales J, Burdett T, Hall P, Junkins H, Klemm A, Flicek P, Manolio T, Hindorff L, Parkinson H. The NHGRI GWAS Catalog, a curated resource of SNP-trait associations. Nucleic Acids Res. 2014 Jan;42(Database issue):D1001-6. doi: 10.1093/nar/gkt1229. Epub 2013 Dec 6. PubMed PMID: 24316577; PubMed Central PMCID: PMC3965119."

[International HapMap Consortium 2003]: http://www.ncbi.nlm.nih.gov/pubmed/14685227 "International HapMap Consortium. The International HapMap Project. Nature. 2003 Dec 18;426(6968):789-96. PubMed PMID: 14685227."

[Nicolae 2010]: http://www.ncbi.nlm.nih.gov/pubmed/20369019 "Nicolae DL, Gamazon E, Zhang W, Duan S, Dolan ME, Cox NJ. Trait-associated SNPs are more likely to be eQTLs: annotation to enhance discovery from GWAS. PLoS Genet. 2010 Apr 1;6(4):e1000888. doi: 10.1371/journal.pgen.1000888. PubMed PMID:  20369019; PubMed Central PMCID: PMC2848547."

[Jostins 2012]: http://www.ncbi.nlm.nih.gov/pubmed/23128233 "Jostins L, Ripke S, Weersma RK, Duerr RH, McGovern DP, Hui KY, Lee JC, Schumm  LP, Sharma Y, Anderson CA, Essers J, Mitrovic M, Ning K, Cleynen I, Theatre E, Spain SL, Raychaudhuri S, Goyette P, Wei Z, Abraham C, Achkar JP, Ahmad T, Amininejad L, Ananthakrishnan AN, Andersen V, Andrews JM, Baidoo L, Balschun T, Bampton PA, Bitton A, Boucher G, Brand S, Büning C, Cohain A, Cichon S, D'Amato M, De Jong D, Devaney KL, Dubinsky M, Edwards C, Ellinghaus D, Ferguson LR, Franchimont D, Fransen K, Gearry R, Georges M, Gieger C, Glas J, Haritunians T, Hart A, Hawkey C, Hedl M, Hu X, Karlsen TH, Kupcinskas L, Kugathasan S, Latiano A, Laukens D, Lawrance IC, Lees CW, Louis E, Mahy G, Mansfield J, Morgan AR, Mowat C, Newman W, Palmieri O, Ponsioen CY, Potocnik U, Prescott NJ, Regueiro M,  Rotter JI, Russell RK, Sanderson JD, Sans M, Satsangi J, Schreiber S, Simms LA, Sventoraityte J, Targan SR, Taylor KD, Tremelling M, Verspaget HW, De Vos M, Wijmenga C, Wilson DC, Winkelmann J, Xavier RJ, Zeissig S, Zhang B, Zhang CK, Zhao H; International IBD Genetics Consortium (IIBDGC), Silverberg MS, Annese V,  Hakonarson H, Brant SR, Radford-Smith G, Mathew CG, Rioux JD, Schadt EE, Daly MJ, Franke A, Parkes M, Vermeire S, Barrett JC, Cho JH. Host-microbe interactions have shaped the genetic architecture of inflammatory bowel disease. Nature. 2012  Nov 1;491(7422):119-24. doi: 10.1038/nature11582. PubMed PMID: 23128233; PubMed Central PMCID: PMC3491803."

[Rivas 2011]: http://www.ncbi.nlm.nih.gov/pubmed/21983784 "Rivas MA, Beaudoin M, Gardet A, Stevens C, Sharma Y, Zhang CK, Boucher G, Ripke S, Ellinghaus D, Burtt N, Fennell T, Kirby A, Latiano A, Goyette P, Green T, Halfvarson J, Haritunians T, Korn JM, Kuruvilla F, Lagacé C, Neale B, Lo KS, Schumm P, Törkvist L; National Institute of Diabetes and Digestive Kidney Diseases Inflammatory Bowel Disease Genetics Consortium (NIDDK IBDGC); United Kingdom Inflammatory Bowel Disease Genetics Consortium; International Inflammatory Bowel Disease Genetics Consortium, Dubinsky MC, Brant SR, Silverberg MS, Duerr RH, Altshuler D, Gabriel S, Lettre G, Franke A, D'Amato M, McGovern DP, Cho JH, Rioux JD, Xavier RJ, Daly MJ. Deep resequencing of GWAS loci identifies independent rare variants associated with inflammatory bowel disease. Nat Genet.  2011 Oct 9;43(11):1066-73. doi: 10.1038/ng.952. PubMed PMID: 21983784; PubMed Central PMCID: PMC3378381."

[Bulik-Sullivan 2015]: http://www.ncbi.nlm.nih.gov/pubmed/25642630 "Bulik-Sullivan BK, Loh PR, Finucane HK, Ripke S, Yang J; Schizophrenia Working Group of the Psychiatric Genomics Consortium, Patterson N, Daly MJ, Price AL, Neale BM. LD Score regression distinguishes confounding from polygenicity in genome-wide association studies. Nat Genet. 2015 Mar;47(3):291-5. doi: 10.1038/ng.3211. Epub 2015 Feb 2. PubMed PMID: 25642630."






